docs/reference/infrastructure/services.md

```diff
@@ -0,0 +1,174 @@
+# Services & providers
```

Comment:
I think we could subdivide this into categories, and everything that is GH based, anaconda related, organization related (teams, repos...) would be under its own group of [tabs ](https://docusaurus.io/docs/next/markdown-features/tabs)

Reply:
Ah wait it does have tabs natively? Wow!

Reply:
On second thought, tabs are not directly linkable via `#anchors`, which might hinder navigation.

Reply:
Oh [it does support it](https://docusaurus.io/docs/markdown-features/tabs?current-os=ios#query-string) via `?query` strings but that's a bit weird :/